nautobot_ssot_aristacv/diffsync/models/nautobot.py

```diff
@@ -54,19 +59,26 @@ def create(cls, diffsync, ids, attrs):
                 role_map[role_code],
                 PLUGIN_SETTINGS.get("from_cloudvision_default_device_role_color", DEFAULT_DEVICE_ROLE_COLOR),
             )
+        elif "CloudVision" in ids["name"]:
+            role = nautobot.verify_device_role_object("Controller", DEFAULT_DEVICE_ROLE_COLOR)
         else:
             role = nautobot.verify_device_role_object(
                 PLUGIN_SETTINGS.get("from_cloudvision_default_device_role", DEFAULT_DEVICE_ROLE),
                 PLUGIN_SETTINGS.get("from_cloudvision_default_device_role_color", DEFAULT_DEVICE_ROLE_COLOR),
             )
 
+        if PLUGIN_SETTINGS.get("create_controller") and "CloudVision" in ids["name"]:
+            platform = OrmPlatform.objects.get(slug="arista_eos_cloudvision")
```

Comment:
would make sesnse to define these well-known slugs in some constant?

Reply:
Sure, I can do that.